supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -1107,12 +1107,16 @@ class GcpBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
       Try {
         runStatus match {
           case _: RunStatus.Aborted => AbortedExecutionHandle
```

Comment:
Is this case still valid since `runStatus` is of type `RunStatus.Failed`?

Reply:
Yes, don't be distracted by the inner method `handleFailedRunStatus` above, this `runStatus` is actually of type `RunStatus` - see line 1089.

Reply:
Ah I see. Thank you for the clarification!